test/e2e/page-objects/secure-your-wallet.ts

```diff
@@ -0,0 +1,53 @@
+import { By, WebDriver } from 'selenium-webdriver'
+import { clickElement, getByDataTestID, isElementDisplayed } from '../selenium-util'
+import * as locators from '../../../src/locator-ids'
+import 'jest-expect-message'
+import componentLocators from '../../../src/components/locators'
+
+export class SecureYourWallet {
+  private readonly revealRecoveryPhraseButton: By = getByDataTestID(componentLocators.mnemonicContainerHidden)
+  private readonly hideRecoveryPhraseButton: By = getByDataTestID(componentLocators.hideIcon)
+  private readonly recoveryPhraseElement: By = getByDataTestID(componentLocators.mnemonicContainerShown)
+  private readonly secureYourWalletPage: By = getByDataTestID(locators.secureYourWalletPage)
+  private readonly copyRecoveryPhraseToClipboardButton: By = getByDataTestID(componentLocators.copyWithCheck)
+  private readonly acknowledgeRecoveryPhraseWarningCheckbox: By = By.id(locators.recoveryPhraseWarningCheckbox)
+  private readonly secureWalletContinueButton: By = getByDataTestID(locators.saveMnemonicButton)
+
+  constructor(private readonly driver: WebDriver) {}
+
+  async revealRecoveryPhrase(acknwledgeWarningAndContinue: boolean = false) {
+    await clickElement(this.driver, this.revealRecoveryPhraseButton)
+    await clickElement(this.driver, this.copyRecoveryPhraseToClipboardButton)
+    if (acknwledgeWarningAndContinue) {
+      await clickElement(this.driver, this.acknowledgeRecoveryPhraseWarningCheckbox)
+      await clickElement(this.driver, this.secureWalletContinueButton)
+    }
+  }
+
+  async hideRecoveryPhrase() {
+    await clickElement(this.driver, this.hideRecoveryPhraseButton)
+  }
+
+  async isRecoveryPhraseDisplayed() {
+    return await isElementDisplayed(this.driver, this.recoveryPhraseElement)
+  }
+
+  async isRecoveryPhraseHidden() {
```

Comment:
This is more for an instant way to check if something is hidden. isRecoveryPhraseDisplayed() above will wait for it to be displayed for a while, adding the negative check when you're expecting a negative outcome just streamlines it a bit